apps/volontulo/views/pages.py

```diff
@@ -1,31 +1,34 @@
 # -*- coding: utf-8 -*-
-# pylint: disable=missing-docstring
+
+"""
+.. module:: pages
+"""
+
 from django.core.urlresolvers import reverse_lazy
 from django.shortcuts import redirect
 from django.views.generic.detail import DetailView
-from django.views.generic.edit import (
-    CreateView, UpdateView, DeleteView
-)
+from django.views.generic.edit import CreateView
+from django.views.generic.edit import DeleteView
+from django.views.generic.edit import UpdateView
```

Comment:
As PEP8 says (https://www.python.org/dev/peps/pep-0008/#imports) every import should be in separate line. It also stand that the way of formatting imports that You propose is OK, but from the beginning of this project we tend to format imports the way I've formatted them here. My experience says that one line per import is the most readable way.